tools/ansible-pr/run.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+set -x
+
+declare -a ignored_modules=(
+  gcp_backend_service
+  gcp_forwarding_rule
+  gcp_healthcheck
+  gcp_target_proxy
+  gcp_url_map
+)
+
+new_branch() {
+  branch_name=$1
+  git  checkout upstream/devel
+  git branch -D $branch_name
+  git chkecout -b $branch_name
+}
+
+get_all_modules() {
+  remote_name=$1
+  file_name=$remote_name
+  git fetch $remote_name
+  git checkout $remote_name/devel
+  git ls-files -- lib/ansible/modules/cloud/google/gcp_* | cut -d/ -f 6 | cut -d. -f 1 > $file_name
+  
+  for i in "${ignored_modules[@]}"; do
+    sed -i "/$i/d" $file_name
+  done
+}
+
+if ! which hub; then
+  echo "Please install the hub CLI"
+  exit 1
+fi
+
+if ! git remote -v | grep "origin"; then
```

Comment:
If this is running in a container, we can set the remote explicitly here rather than requiring it to already be set up.

Reply:
That's true.

Reply:
For instance, I don't use an origin remote - so this script isn't good on my machine.  It seems like it might be better not to make assumptions about the remotes in question.